src/endout/test/endout.test.cpp

```diff
@@ -12,6 +12,7 @@ using namespace njoy::ENDFtk;
 using IncoherentElastic = section::Type<7,2>::IncoherentElastic;
 using MF7 = njoy::ENDFtk::file::Type<7>;
 
+/*
```

Comment:
Was this intended to be commented out?

Reply:
Oh heavens. You're right I can't believe I forgot about this. This test I believe needs to be rechecked since I changed the inputs to endout. Thank you for pointing this one out